### DISCUSSION TITLE
Mac 10.15 / Ableton Live 11

### DISCUSSION THREAD

**Discussion by 24Skore** [ZL-Audio/ZLEqualizer#29] [General]
Hello and congratulations on your audio plug-in development work.
I just created a GitHub account as I wanted to contact you with some feedback that unfortunately I cannot open ZL Equalizer 0.1.12 on my system.
I use MacOS 10.15 Catalina and Ableton Live 11 and I have manually installed the VST3 & AU files to Mac HD/Library/Audio/Plugins (VST3 & Component folders) this is where all of my other audio plug-ins are installed.

When I open Ableton Live, I get a standard Apple Gatekeeper message:
“ZL Equalizer.vst3” can’t be opened because Apple cannot check it for malicious software.
I open System Preferences - Security & Privacy and click “open anyway” but in Ableton Live, only The AU Component file shows up in the plugin menu, but I can't open it to use it as I see the text:
This audio unit v2 plug-in could not be opened.

I hope this info helps you with future development, also ZL Equalizer has been featured on Bedroom Producers Blog where you might find some more useful feedback as well
https://bedroomproducersblog.com/2024/03/22/zl-equalizer/

Thanks so much and keep up your great work.

**Comment by zsliu98** [comment]
It seems that you have the same issue as #24 . Since I do not have a macOS 10.15 machine and GitHub only provides macOS 11/12/13 runners, I am not able to debug it for you. I am afraid that the most practical solution is to upgrade the OS at this moment.
I may open a branch later to test this :) stay tuned

**Reply by 24Skore** [reply]
Thanks for your reply, I didn't realise there was another similar comment.
I am stuck on Catalina for now as this Mac has reached its final update, thanks for your advice though, I see many plugins with MacOS 11 as the minimum system requirements these days, so I can understand your limitations as well.
Thanks

**Comment by zsliu98** [comment]
Hi! If you have time, would you like to check whether the artifact from the [workflow](https://github.com/ZL-Audio/ZLEqualizer/actions/runs/8439522945) is compatible with your OS version (macOS 10.15)? (malicious software warning will persist unless you build the source yourself). If you cannot access the artifact, please tell me.

**Reply by 24Skore** [reply]
I clicked the link and I can see this: "ZL Equalizer-0.0.0.dmg" but I can't click the text to download it.
I had a look here: https://github.com/ZL-Audio/ZLEqualizer/releases?page=3 but there the earliest version is 0.0.1
Thanks

**Reply by zsliu98** [reply]
Never mind. I will attach the zip file here. Version 0.0.0 indicates that it is not a formal release.
[ZL Equalizer-0.0.0.dmg.zip](https://github.com/ZL-Audio/ZLEqualizer/files/14765268/ZL.Equalizer-0.0.0.dmg.zip)

**Reply by 24Skore** [reply]
Thanks, i will try now

**Reply by 24Skore** [reply]
Hi, sorry to say that it's still mot working. I get the same message in Ableton Live 11 for the AU Component file as before and the same result with the VST3 not showing up in the plugin menu or the search results.

Many thanks for your help and for your time as well, it is very much appreciated that you give your link for me to try, and it was well worth trying to see if it would work as well as you have created an awesome eq.
I wish you well for your future, keep up your great work as many other people will really appreciate what you are doing. I saw someone left a comment on BPB (Bedroom Producers Blog) that they wanted to nominate ZL Equalizer as plugin of the year, so well done !
All the best ;0)

**Reply by zsliu98** [reply]
I'm sorry to hear that it still does not work :pensive: I am currently working on some other issues. If someday I find a solution, I will also update here. I appreciate your feedback and support.

**Reply by 24Skore** [reply]
It's ok, I hope to update my computer soon.
Your welcome and I wish you good luck with your other issues and thanks for everything too.

All the best

**Comment by leonsmits479** [comment]
Hi, I have the same problem with Mac OS BigSur version 11. Cubase did not recognise the VST3.
Someone some tips what i can do?

Kind Regards leon
